### PR TITLE
Kustomer plugin

### PR DESCRIPTION
**Pull request by mknippen**
Version 0.2.0 offers many new features:

1. The ability for users to hide and show the Forethought support view without having to manage the view controller
2. Support for plugins, where you can extend Forethought to show your live chat platforms with very little code
3. Created the first Forethought Plugin for Kustomer

This version also has many other enhancements and fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,11 @@ You will need a valid API key in order to use the Forethought Solve SDK. In addi
 1. Forethought is also available through [CocoaPods](http://cocoapods.org). If you have not yet installed Cocoapods, follow their guide to install and create a Podfile [here](https://guides.cocoapods.org/using/getting-started.html).  
 
 1. In your Podfile, add the following line:
+<<<<<<< HEAD
+   ```ruby
```
Comment (sbastidasr) [inline README.md:23]: merge conflict

File: ForethoughtPlugins.podspec [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  spec.name         = "ForethoughtPlugins"
+  spec.version      = $FORETHOUGHT_VERSION
+  spec.summary      = "Connect Forethought to various other live-chat platforms"
+  spec.description  = "Connect Forethought to various other live-chat platforms, such as ZenDesk or Kustomer"
+  spec.homepage     = "https://forethought.ai"
+  # spec.screenshots  = "www.example.com/screenshots_1.gif", "www.example.com/screenshots_2.gif"
+  spec.license      = { :type => "Apache License, Version 2.0", :file => "LICENSE" }
+
+  spec.author             = { "Matthew Knippen" => "matthew@zwiffer.com" }
```
Comment (sbastidasr) [inline ForethoughtPlugins.podspec:13]: `eng@forethought.ai`  🙏

File: README.md (hunk 1 above)
Comment (mknippen) [reply README.md:22]: fixed

**Review by sbastidasr — APPROVED** [review]
⚡

**Pull request merged**